features/works/work_download.feature

```diff
@@ -5,180 +5,172 @@
 Feature: Download a work
   @wip
```

Comment:
It looks like taking `@wip` out of the file leads to the tests running and thus failing, so I've put it back in and maybe some brave soul will be able to finish this some day.